### ISSUE TITLE
Legal entities list/by id for nhs admin v2

### ISSUE THREAD

**Issue by danakonovalenko** [edenlabllc/ehealth.api#3116]
NHS admin must be able to get legal entities list using filter parameters and sort parameters which described in [scheme](
https://github.com/edenlabllc/ehealth.web/blob/master/packages/mock-server/__schema__/legal_entities.graphql) and [features](
https://github.com/edenlabllc/ehealth.web/blob/master/packages/admin/e2e/__features__/legal_entities_details.feature)
-`legalEntity(id: ID!): LegalEntity`
-`legalEntities(filter: LegalEntityFilter, orderBy: LegalEntityOrderBy): LegalEntityConnection!`

**Comment by danakonovalenko** [comment]
- [ ] [test cases](https://edenlab.atlassian.net/wiki/spaces/EH/pages/668369088/Get+Legal+Entities+list+ID)

**Comment by danakonovalenko** [comment]
archive can be null array. As for now I receive an error 
```
{
	"data": {
		"legalEntity": null
	},
	"errors": [
		{
			"locations": [
				{
					"column": 0,
					"line": 6
				}
			],
			"message": "Cannot return null for non-nullable field",
			"path": [
				"legalEntity",
				"archive"
			]
		}
	]
}
```

**Comment by danakonovalenko** [comment]
I still receive same error when archive is null

**Comment by danakonovalenko** [comment]
we need to change the logic for retiring owner object. As for now we return `employee` with  `employee_type=OWNER and is_active=true`. However we should return  first`employee ` with  `employee_type=OWNER order by updated_at_desc limit 1`

**Comment by danakonovalenko** [comment]
- [ ] filter by databaseId - 500

**Comment by PavelVesnin** [comment]
fixed on dev #3483
